content_arfs/code/mob/pokemon.dm

```diff
@@ -456,6 +456,15 @@
 	movement_cooldown = 1.5
 	has_hands = TRUE
 
+/mob/living/simple_mob/animal/passive/pokemon/leg/lugia/andy
+	health = 500
+	maxHealth = 500
+	vore_capacity = 2
+	size_multiplier = 2
+	default_pixel_y = 16
+	pixel_y = 16
+	old_y = 16
+
```

Comment:
legendary mons should be offset by -32! (the base legendary mob already has this)

Reply:
Yes, they have a -32 _x_ offset. This is a _y_ offset. Because the offset needed to make it look right (or at least how I wanted it) changes a bit when you scale up.

Reply:
oh derp, read that too fast